sys-apps/man-db/man-db-2.7.6.1-r3.ebuild

```diff
@@ -12,7 +12,7 @@ SRC_URI="mirror://nongnu/${PN}/${P}.tar.xz"
 LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="alpha amd64 arm ~arm64 hppa ia64 ~m68k ~mips ppc ppc64 ~s390 ~sh sparc x86 ~amd64-linux ~arm-linux ~x86-linux"
-IUSE="berkdb +gdbm +manpager nls selinux static-libs zlib"
+IUSE="berkdb +cron +gdbm +manpager nls selinux static-libs systemd zlib"
```

Comment:
The problem is not the installation of a small file. If I'm using an still mixed cron/systemd-timer system the installation of the cron files will have the side effect to automatically enable twice the activities: one from cron and one from systemd-timer. If I don't want cron files I should avoid to install these files.